backend/cmm.ml

```diff
@@ -218,11 +218,9 @@ and operation =
   | Copaque
   | Cbeginregion | Cendregion
 
-type value_kind =
-  | Vval of Lambda.value_kind (* Valid OCaml values *)
-  | Vint (* Untagged integers and off-heap pointers *)
-  | Vaddr (* Derived pointers *)
-  | Vfloat (* Unboxed floating-point numbers *)
+type kind_for_unboxing =
+  | Any
```

Comment:
Could you add a comment stating that this is anything ? (e.g. non-scannable thing, mix of scannable and non-scannable thing), etc...